Userland/Libraries/LibAudio/ConnectionToManagerServer.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "ConnectionToManagerServer.h"
+
+namespace Audio {
+
+ConnectionToManagerServer::ConnectionToManagerServer(NonnullOwnPtr<Core::LocalSocket> socket)
+    : IPC::ConnectionToServer<AudioManagerClientEndpoint, AudioManagerServerEndpoint>(*this, move(socket))
+{
+}
+
+ConnectionToManagerServer::~ConnectionToManagerServer()
+{
+    die();
+}
+
+void ConnectionToManagerServer::die() { }
+
+void ConnectionToManagerServer::main_mix_muted_state_changed(bool muted)
+{
+    if (on_main_mix_muted_state_change)
+        on_main_mix_muted_state_change(muted);
+}
+
+void ConnectionToManagerServer::main_mix_volume_changed(double volume)
+{
+    if (on_main_mix_volume_change)
+        on_main_mix_volume_change(volume);
+}
+
+void ConnectionToManagerServer::hardware_sample_rate_changed(u32 sample_rate)
```

Comment:
Terminology: maybe instead of "hardware", let's call it "device" so it doesn't imply anything about the implementation of the audio sink.